File: RepoTransBench/source_projects/Python/initstring_linkedin2username/tests/test_georegions.py
import pytest
from linkedin2username import GEO_REGIONS

def test_geo_regions_us():
    assert GEO_REGIONS["us"] == "103644278"

def test_geo_regions_all_have_str():
    for code, val in GEO_REGIONS.items():
        assert isinstance(code, str)
        assert isinstance(val, str)
        assert val.isdigit()

File: RepoTransBench/target_projects/Python/Rust/initstring_linkedin2username/tests/original/test_georegions.rs
use linkedin2username::GEO_REGIONS;

#[test]
fn test_geo_regions_us() {
    let geo = GEO_REGIONS.lock().unwrap();
    assert_eq!(geo.get("us").unwrap(), "103644278");
}

#[test]
fn test_geo_regions_all_have_str() {
    let geo = GEO_REGIONS.lock().unwrap();
    for (code, val) in geo.iter() {
        assert!(code.is_ascii() || !code.is_empty()); // code is always String
        assert!(val.is_ascii() || !val.is_empty());
        assert!(val.chars().all(|c| c.is_ascii_digit()), "All values must be numeric string");
    }
}